guillotina/db/storages/pg.py

```diff
@@ -539,26 +546,26 @@ def lock(self):
     async def create(self):
         # Check DB
         log.info('Creating initial database objects')
-        statements = [
+
+        statements = []
+
+        if self._db_schema:
+            statements.extend([f'CREATE SCHEMA {self._db_schema}'])
+
+        statements.extend([
             get_table_definition(self._objects_table_name, self._object_schema),
             get_table_definition(self._blobs_table_name, self._blob_schema,
                                  primary_keys=('bid', 'zoid', 'chunk_index'))
-        ]
+        ])
         statements.extend(self._initialize_statements)
 
         for statement in statements:
-            otable_name = self._objects_table_name
```

Comment:
Unfortunately, this was a b/w compat way to not recreate indexes. It rewrites the default `objects` -> `object` and `blobs` -> `blob`.

So unfortunately, we still need something like this around... :(

Reply:
I moved the logic here https://github.com/plone/guillotina/pull/425/files/edf11f986aefdf4139891c543bd2b414b7648316#diff-d558ee25390b1019470ae02f3f71861bR582